protected/humhub/modules/comment/Module.php

```diff
@@ -37,7 +37,7 @@ public function getPermissions($contentContainer = null)
     {
         if ($contentContainer instanceof Space) {
```

Comment:
In light of #4738 it would be better to remove `instanceof Space` and open the permissions up to also profile comments as well as Space, your thoughts @yurabakhtin?

Reply:
Yes, it is strange why the permission was developed only for Space, because I see we can create comments in User container as well, so no reason to keep this only for Space. But I didn't do this so I cannot change this right now, maybe @luke- or @buddh4 know this code better.

Reply:
@ArchBlood thank you for your PR here!